k8s-manifests/README.md

```diff
@@ -45,27 +45,29 @@ helm template redpanda redpanda/redpanda \
 >Then, the `--reserve-memory` needs to be reduced from 205 to 0M.
 >After that, the `spec.template.spec.containers.resources.limits.memory` should be reduced from 2.5Gi to 1Gi
 
-3. Copy the Statefulset, `redpanda` and `redpanda-external` services, and ConfigMap to the Datacater kubernetes manifest.
+3. Copy the StatefulSet, the Services `redpanda` and `redpanda-external`, and the ConfigMap to DataCater's Kubernetes manifest, which you generated as describe above.
 
-## working with Redpanda in Cluster
-To access the broker inside datacater, the `bootsrap.servers` 
```

Comment:
`bootsrap.servers` -> `bootstrap.servers`

Reply:
Thank you for catching it. I fixed it :)